### ISSUE TITLE
Add Standings 

### ISSUE THREAD

**Issue by MrTob** [openfootball/euro.json#1]
The results of the group phase are still partially undefined (null).
e.g 
```json
 {
          "num": 32,
          "date": "2021-06-15",
          "team1": {
            "name": "France",
            "code": "FRA"
          },
          "team2": {
            "name": "Germany",
            "code": "GER"
          },
          "score1": null,
          "score2": null,
          "score1i": null,
          "score2i": null,
          "group": "Group F"
        }

**Comment by geraldb** [comment]
@MrTob   good catch. it is day 3 on the restart.  i am rebuilding and reorganizing in the next days the underlying machinery, see here <https://github.com/sportdb>.  more euro.json updates for sure in the next weeks. thank you for your patience.

**Comment by geraldb** [comment]
@MrTob   please check again - i pushed an update for euro 2020 (in 2021) now with all matches and results (plus goals), see <https://github.com/openfootball/euro.json/blob/master/2020/euro.json> 
ps: the data source is here <https://github.com/openfootball/euro/blob/master/2020--europe/euro.txt>
